janitor/functions.py

```diff
@@ -4055,6 +4264,89 @@ def take_a_if_even(x):
     return df
 
 
+@pf.register_dataframe_method
+def join_conditional(
+    df: pd.DataFrame, left_column: str, right_column: str, join_operator: str
+) -> pd.DataFrame:
+    """
+    Create a new DataFrame based on the
+    comparison applied using the parameters listed
+
+    This method does not change the original DataFrame.
+
+     This function is supposed to emulate SQL and R's
```

Comment:
From this line onwards, the indentation appears to be off. Can you make sure the function docstring's indentation is consistent?

Reply:
**Action item:** Please ensure indentation is correct. At the end, please also check that the function documentation is visible on the docs preview. Ping us if you are unsure how to check the docs preview! :smile: